recipe/meta.yaml

```diff
@@ -9,19 +9,22 @@ source:
   sha256: 7f6b2733bde06ea2575b79a6450d9bd5c38918ff4cb2a355bf22bbe8c86c6bcf
 
 build:
+  # skip py<39 required from upstream
```

Comment:
remove that comment, we suppose to be always driven by upstream according to python version supported

Reply:
I'd like @skupr-anaconda to chime in on this one, because I started doing this after this was done by him (no blaming intended, maybe I misunderstood something).

Reply:
I would put the same comment to anyone, but sure lets wait for Serhii

Reply:
Oh sorry, it was really just about the comment. Sometimes when I think of a selector my imagination still calls it "comment".